src/connectedk8s/azext_connectedk8s/_troubleshootutils.py

```diff
@@ -868,3 +868,248 @@ def describe_non_ready_agent_log(filepath_with_timestamp, corev1_api_instance, a
             diagnoser_output.append("An exception has occured while storing stuck agent logs in the user local machine. Exception: {}".format(str(e)) + "\n")
 
     return storage_space_available
+
+
+def get_secrets_azure_arc(corev1_api_instance, kubectl_client_location, kube_config, kube_context, filepath_with_timestamp, storage_space_available):
+
+    try:
+        if storage_space_available:
+            command = [kubectl_client_location, "get", "secrets", "-n", "azure-arc"]
```

Comment:
did you check if there was a python sdk method to do this? we should use kubectl only when python sdk method is not available

Reply:
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CoreV1Api.md#list_namespaced_secret

Reply:
example from our repo - https://github.com/Azure/azure-cli-extensions/blob/6efefd132630941c8277368d604b30e1793322b0/src/connectedk8s/azext_connectedk8s/_troubleshootutils.py#L124

Reply:
there is an sdk method for secrets but but , it gives the complete details of the secrets including private key uid  client id and more , that is why i chose the route of kubectl way so as to only retrieve the names and age of the secret